apps/web/src/pages/templates/TemplatesListPage.tsx

```diff
@@ -16,29 +16,27 @@ import { Data } from '../../design-system/table/Table';
 import { ROUTES } from '../../constants/routes.enum';
 import { parseUrl } from '../../utils/routeUtils';
 import { TemplatesListNoData } from './TemplatesListNoData';
-import { useCreateDigestDemoWorkflow } from '../../api/hooks/notification-templates/useCreateDigestDemoWorkflow';
 import { useSegment } from '../../components/providers/SegmentProvider';
 import { TemplateAnalyticsEnum } from './constants';
-/*
- * TODO uncomment when will be using templates store
- * import { useTemplatesStoreModal } from './hooks/useTemplatesStoreModal';
- */
+import { useTemplatesStoreModal } from './hooks/useTemplatesStoreModal';
+import { useFetchBlueprints } from '../../api/hooks/notification-templates/useFetchBlueprints';
 
 function NotificationList() {
   const segment = useSegment();
   const { readonly } = useEnvController();
   const [page, setPage] = useState<number>(0);
-  const { groups, loading: areNotificationGroupLoading } = useNotificationGroup();
+  const { loading: areNotificationGroupLoading } = useNotificationGroup();
   const { templates, loading: isLoading, totalCount: totalTemplatesCount, pageSize } = useTemplates(page);
   const theme = useMantineTheme();
   const navigate = useNavigate();
+  const {
+    blueprintsGroupedAndPopular: { groupedBlueprints, popularBlueprints } = {},
+    isLoading: areBlueprintsLoading,
+  } = useFetchBlueprints();
+  const hasGroups = groupedBlueprints && groupedBlueprints.length > 0;
+  const hasTemplates = templates && templates.length > 0;
```

Comment:
```suggestion
  const hasGroups = groupedBlueprints?.length > 0;
  const hasTemplates = templates?.length > 0;
```
🏌🏻 But a verbose approach is great on my book too.

Reply:
`'groupedBlueprints.length' is possibly 'undefined'`, not sure why it doesn't like it, maybe because of the `undefined > 0` not sure